### ISSUE TITLE
High loss_value dur to High confidence_loss

### ISSUE THREAD

**Issue by desmanda** [qqwweee/keras-yolo3#164]
Hi,
I am using this code to train it on a custom dataset ( two classes). I have a very high loss_value, around 30 sometimes 50. After debugging, I found out that this is due to a **high confidence loss**.
Do you have any idea how to fix this issue?
Thanks!

**Comment by damvantai** [comment]
i think High loss value not importance. Importance is accuracy in test dataset and run in real

**Comment by damvantai** [comment]
are you implement fine tune epoch 50 - 100 (request many memory GPU)?

**Comment by desmanda** [comment]
yeah, I am using the same fine-tuning as in the current code, but I resized the images as well to 320 to fix the issue with the memory GPU and the batch size to 24.

**Comment by MichalDulemba** [comment]
Mine goes to around 16 and stays there. Also with fine tuning. Any ideas?

**Comment by OtiTantaoui** [comment]
I have the same problem here, I tried using different learning rate, tried to fine tune and defreez all network but any way the loss is stuck at around 30. If someone found the solution please notify me. or even if you found an other repository that allowed you to train on your data without problem, please let me know

**Comment by desmanda** [comment]
Hi, from my experience I would rather not rely at all on the loss value. I had good mAP while my loss value was around 12. It really depends on the number of objects in one image and the dataset. So the next step will be to evaluate your model using mean average precision or simply precision and recall for a given threshold score and IOU. Before that, you should also have a look to ur labels, maybe you have some wrong labels ( it happened to me). Good luck!

**Comment by eain3314** [comment]
我的损失很高 测试效果也不理想怎么办
